src/data.ts

```diff
@@ -52,3 +52,141 @@ export const FAQData: { question: string; answer: string }[] = [
   },
 ];
+
+export const codeOfConductText: {
+  paragraph: string[];
+  title: string;
+}[] = [
+  {
+    title: "Code of Conduct",
+    paragraph: [
+      "All participants of CUSEC are expected to abide by our Code of Conduct, both online and during in-person events that are hosted and/or associated with CUSEC.",
+    ],
+  },
+  {
+    title: "The Pledge",
+    paragraph: [
+      "In the interest of fostering an open and welcoming environment, we pledge to make participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.",
+    ],
+  },
+  {
+    title: "The Standards",
+    paragraph: [
+      "Examples of behaviour that contributes to creating a positive environment include: ",
+      "• Using welcoming and inclusive language. ",
+      "• Being respectful of differing viewpoints and experiences. ",
+      "• Gracefully accepting constructive criticism. ",
+      "• Referring to people by their preferred pronouns and using gender-neutral pronouns when uncertain. ",
+      "Examples of unacceptable behaviour by participants include: ",
+      "• Trolling, insulting/derogatory comments, public or private harassment. ",
+      "• Publishing others' private information, such as a physical or electronic address, without explicit permission. ",
+      "• Not being respectful to reasonable communication boundaries, such as 'leave me alone,' 'go away,' or 'I’m not discussing this with you.' ",
+      "• The usage of sexualised language or imagery and unwelcome sexual attention or advances. ",
+      "• Swearing, usage of strong or disturbing language. ",
+      "• Demonstrating the graphics or any other content you know may be considered disturbing. ",
+      "• Starting and/or participating in arguments related to politics. ",
+      "• Assuming or promoting any kind of inequality including but not limited to: age, body size, disability, ethnicity, gender identity and expression, nationality and race, personal appearance, religion, or sexual identity and orientation. ",
+      "• Drug promotion of any kind. ",
+      "• Attacking personal tastes. ",
+      "• Other conduct which you know could reasonably be considered inappropriate in a professional setting.",
```

Comment:
Nitpick: consider adding some spacing/indentation on the left of bullet points (it may not worth it since it looks difficult to style individual text entries)

Reply:
I can do this by checking the first character and add a non-breaking space if it is a bullet